lib/tasks/gather.js

```diff
@@ -736,6 +736,18 @@ class TaskGather extends SttTask {
       this.logger.info({evt}, 'TaskGather:_onTranscription - got empty transcript, continue listening');
       return;
     }
+    const confidence = evt.alternatives[0].confidence;
+    const minConfidence = this.data?.recognizer?.minConfidence;
+    this.logger.debug({evt},
+      `TaskGather:_onTranscription - confidence (${confidence}), minConfidence (${minConfidence})`);
+    if (confidence && minConfidence && confidence < minConfidence) {
+      this.logger.info({evt},
+        'TaskGather:_onTranscription - rejecting transcript as confidence ' +
+        `(${confidence}) < minConfidence (${minConfidence})`);
+      evt.error = 'Rejecting transcript as confidence ' +
+        `(${confidence}) < minConfidence (${minConfidence})`
+      return this._resolve('stt-min-confidence-error', evt);
```

Comment:
could we rename this event as `stt-low-confidence`?  Also, I don't really consider this an "error", as nothing necessarily went wrong.

Reply:
Yes, renamed stt-min-confidence-error as stt-min-confidence